examples/basic/variables.tf

```diff
@@ -65,13 +61,13 @@ variable "transit_gateway_name" {
 variable "reuse_cloud_connections" {
   description = "When true, IBM Cloud connections are reused (if attached to the transit gateway)."
   type        = bool
-  default     = true
+  default     = false
 }
 
 variable "cloud_connection_count" {
   description = "Required number of Cloud connections to create or reuse. The maximum number of connections is two per location."
   type        = number
-  default     = 0
+  default     = 2
```

Comment:
should we just use `1` for the basic example?

Reply:
Its okay we can use 2. no harm :)